src/routes/at-resources.routes.js

```diff
@@ -13,7 +13,8 @@ const { dashboard,
         renderAddSprintForm,
         addSprint,
         renderEditSprintForm,
-        updateSprint} = require("../controllers/at-resources.controller");
+        updateSprint,
+        renderMentorList,} = require("../controllers/at-resources.controller");
```

Comment:
should be `renderMentorList} = require("../controllers/at-resources.controller");`

Reply:
Correction ready